python/pyhdk/hdk.py

```diff
@@ -1853,7 +1856,30 @@ def import_parquet(self, file_name, table_name=None, fragment_size=None):
         QueryExpr
             Scan expression referencing created table.
         """
-        pass
+        files = self.__verify_files_param(file_name)
+
+        table_opts = TableOptions()
+        if fragment_size is not None:
+            table_opts.fragment_size = fragment_size
+
+        real_table_name, append = self._process_import_table_name(table_name)
+        # Create new table by importing the first file or simply check its schema
+        # matches the specified one.
+        if append:
+            # check schema
+            pass
+        else:
+            self._storage.importParquetFile( files[0], real_table_name, table_opts)
```

Comment:
Looks like the code is not formatted. We do not force Python code formatting in CI, but please apply `black` formatter here.